db/migrate/20211230202951_create_last_wrangling_activities.rb

```diff
@@ -0,0 +1,12 @@
+class CreateLastWranglingActivities < ActiveRecord::Migration[5.2]
```

Comment:
Can you update this to 6.0? 5.2 defaults to timestamps precise down to the second, whereas I think 6.0 defaults to timestamps precise down to the microsecond.

(I was going to ask you to add a `travel_to` call back to the `not_to change` tests, since those could cause misleading results if the first action and the second action occur during the same second. But I don't think you need to change those tests again if you have more precise timestamps, and since you're creating a new table, you can have timestamps as precise as you'd like.)